ArduCopter/radio.cpp

```diff
@@ -91,9 +91,14 @@ void Copter::init_rc_out()
 // enable_motor_output() - enable and output lowest possible value to motors
 void Copter::enable_motor_output()
 {
-    // enable motors
-    motors->enable();
-    motors->output_min();
+    // enable motors. We don't enable if in oneshot mode till we've
+    // completed setup(). This prevents us sending pulses at a slow
+    // rate
+    if (motors->get_pwm_type() < AP_Motors::PWM_TYPE_ONESHOT ||
+        ap.initialised) {
```

Comment:
@tridge @rmackay9 

Tested as is and the issue disappeared.

But also tested with a simpler version of line 97 like

if (ap.initialised) {

and the results were the same.

Major difference is that now the beeping ESC's do when initialised happen at the same time for all variants.

Tested ESC cal on PWM, OneShot, Oneshot 125 and consistent (good) behaviour, so if nothing against I would suggest to apply this for all ESC types

NB: don't have brushed motors :)